external/elemental.cmake

```diff
@@ -167,9 +190,26 @@ if(ENABLE_ELEMENTAL AND DEFINED ELEMENTAL_TAG)
     message (STATUS "** Done configuring Elemental")
   endif(error_code)
 
-  set(${ELEMENTAL_PACKAGE_NAME}_DIR ${ELEMENTAL_BINARY_DIR})
-  find_package(${ELEMENTAL_PACKAGE_NAME} REQUIRED
-               COMPONENTS REQUIRED ${ELEMENTAL_PACKAGE_NAME} pmrrr ElSuiteSparse)
+  execute_process(
+      COMMAND mkdir 
+      ARGS
+      -p
+      ${ELEMENTAL_INSTALL_DIR}/CMake
+      WORKING_DIRECTORY "${ELEMENTAL_INSTALL_DIR}"
+      RESULT_VARIABLE error_code)
+
+  execute_process(
```

Comment:
want to avoid calling unix commands directly. Can you use file(COPY ...)
https://cmake.org/cmake/help/v3.2/command/file.html